src/test/java/net/minecraftforge/debug/ConfigTest.java

```diff
@@ -99,4 +133,48 @@ public SubCat(String value)
             }
         }
     }
+
+    public static class ConfigurationGui extends GuiConfig
```

Comment:
Here is my proposed changes to restore binary compatibility, but I'm not sure if it would work, nor do I know how to test it:
[Gist - Changes](https://gist.github.com/thilokru/8f2bcc20c3c88cf357bce678619bf223)
Would it work? If not, why? What needs to be changed?